test/runtests.jl

```diff
@@ -207,3 +207,23 @@ end
         end
     end
 end
+
+@threads for i = 1:N
+    @testset "wigner9j: relation to sum over 6j products, thread $i" begin
+        for k = 1:10_000
+            @testset let (j1, j2, j3, j4, j5, j6, j7, j8, j9) = rand(smalljlist, 9)
```

Comment:
Maybe replace this one with 
```@testset begin
let ...
```

to make Julia 1.6 happy. However, I am happy to bump the minimum Julia version.

Reply:
Oops - can do! I should have paid attention to that compat check; my bad - that made failing test messages a bit more readable, but I see no need to regress for that.